doc/articles/getting-started-tutorial-1.md

```diff
@@ -225,7 +227,9 @@ The tutorial walks you through creating a cross platform application with Uno, w
     }
     ```
 
-1. Now that we have some basic data to bind to, in the **Solution Explorer**, double-click **MainPage.xaml** to open, then add the following code. We will start by adding a XML Namespace for the Converters and Controls from the Microsoft Community Toolkit as shown below:
+1. Now that we have some basic data to bind to, in the **Solution Explorer**, double-click **MainPage.xaml** to open it.
+1. At the top left corner of the editor select **BugTracker.UWP**
```

Comment:
Maybe adding a screenshot it's helpful in this case ;) ?
![image](https://user-images.githubusercontent.com/16295702/71028686-cd267e00-20db-11ea-88a6-0653a6bfd2af.png)